cli/scorecard/violations.go

```diff
@@ -30,98 +30,99 @@ import (
 	"github.com/forseti-security/config-validator/pkg/api/validator"
 )
 
-func addDataFromReader(config *ScoringConfig, reader io.Reader) error {
+func addDataFromReader(config *ScoringConfig, reader io.Reader) ([]*validator.Asset, error) {
```

Comment:
Since we're changing these functions to get and return the data instead of adding it directly, could we change the names?

```suggestion
func getDataFromReader(config *ScoringConfig, reader io.Reader) ([]*validator.Asset, error) {
```

Reply:
To be clear, please rename all the updated methods to follow this.

Reply:
Oh, I thought that the suggestion was going to update all callsites. will fix.